cmd/dockerd/daemon.go

```diff
@@ -111,6 +111,12 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 		}
 	}
 
+	if cli.LogConfig.Timezone != "" {
```

Comment:
I can't pass `LogConfig` to logger.ValidateLogOpts because at line 178 in daemon/logs.go `daemon/config`.LogConfig and at line 109 in `cmd/dockerd/daemon.go` is passing `api/types/container`.LogConfig. So I am going to add another argument.

Reply:
hmm, I don't understand. Either way don't you have to change the signature of the function?